frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx

```diff
@@ -123,14 +128,22 @@ const DistributeAssessmentPage = (): React.ReactElement => {
             Next
           </Button>
         ) : (
-          <Button
-            leftIcon={<PaperPlaneOutlineIcon />}
-            minWidth="10"
-            onClick={() => console.log("Distribute assessment.")}
-            variant="primary"
-          >
-            Distribute
-          </Button>
+          <>
+            {startDate && endDate && teacherId && (
+              <DistributeAssessmentButton
+                testSessionRequest={{
+                  test: testId,
+                  teacher: teacherId,
+                  school: "639151a4d46e8c002a49f8d6",
```

Comment:
Right now, each `TestSession` has a `school` field. But I have difficulty here populating this field, because we don't actually link `teacher` to `school`. I would have to add a new query `GET_SCHOOL_BY_TEACHER` (and check each school's array of teachers if a given `teacher` belongs to it). 

But I think this begs the question of whether we should just be storing the `school` in the `teacher` object to begin with? i.e why do we store the `school` as a reference in the `test session` object but not in the `teacher` object?

Reply:
Based on Slack convo, I'll add `school` field to `teacher`, but I'll do so in a separate PR so the scope of this one doesn't grow too large! for now, the value will be hardcoded.